src/dev/impl/DevToys/Assets/Icons/Gzip.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   viewBox="0.008 0.008 31.984 31.984"
+   version="1.1"
+   id="svg66"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs29" />
+  <path
```

Comment:
Interestingly enough, this icon doesn't seem to work (there's no icon for me).

What about using a font icon with the glyph `\uF435` ? It's the icon for Zip file.
Here is an example:
https://github.com/veler/DevToys/blob/8e34d4aa4bb0967fdba7a8165d473a99e8d2a708/src/dev/impl/DevToys/ViewModels/Tools/Generators/HashGenerator/HashGeneratorToolProvider.cs#L30

Reply:
That worked fine, finally 👍